src/proc/y411-converter.h

```diff
@@ -0,0 +1,35 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "synthetic-stream.h"
+
+namespace librealsense
+{
+    class y411_converter : public functional_processing_block
+    {
+    public:
+        y411_converter(const rs2_format & target_format)
+            : functional_processing_block("Y411 Transform", target_format) {};
+
+    protected:
+        y411_converter(const char * name, const rs2_format & target_format)
```

Comment:
rs2_format is an enum... why the const &?

Reply:
done